doc/Language/variables.pod6

```diff
@@ -637,20 +636,20 @@ dangerous.
 
 =head3 The C<$> Variable
 
-As well as explicitly declared named state variables C<$> can be used
+Beside explicitly declared named state variables, C<$> can be used
 as an anonymous state variable without an explicit C<state> declaration.
 
     say "1-a 2-b 3-c".subst(:g, /\d/, {<one two three>[$++]});
     # OUTPUT«one-a two-b three-c␤»
 
-Furthermore, state variables are not required to exist in subroutines. You
+Furthermore, state variables are not required in subroutines. You
```

Comment:
This section of the documentation is not about declaring static variables outside of subroutines. It's about how...(a few lines later):
`Each reference to C<$> within a lexical scope, in effect, is a separate
variable.`

Your Perl 6 is stronger than mine, so let me know...